Added Data Analysis notebook and Salts

kassabry opened the pull request:
Adding the raw data analysis notebook so I don't forget, and the Salts file. This is so we have evidence for the numbers and preprocessing we did for the PstP screen

kassabry committed:
Uploaded first round of the data analysis notebook

This is to track the progress of the data analysis notebook so we have a notebook to point to for our data regarding the PstP screen --- I will cleanup and update as more data analysis is done

kassabry committed:
Added the salts file

This is adding the Salts file used to de-salt the molecules so we have a record of how the molecules were preprocessed

agitter commented:
Before merging, can you please give the notebook a more specific name? We may not remember later what `Data Analysis Notebook.ipynb` was used for among the many notebooks.

kassabry committed:
Uploaded CEM Heatmap figures

Put all the CEM heatmap figures in an additional supplementary figures folder, in case we want to add other figures straight to the GitHub

kassabry committed:
Rename Data Analysis Notebook.ipynb to PstP_Screening_Data_Analysis.ipynb
